horizons/world/managers/productionfinishediconmanager.py

```diff
@@ -152,4 +154,4 @@ def get_resource_string(self, instance, res):
 		This key MUST be unique!
 		"""
 		return "produced_resource_" + str(res) + "_" + str(instance.position.origin)\
-		       + "_" + str(Scheduler().cur_tick)
+		       + "_" + str(uuid.uuid1())
```

Comment:
Thing is, and that's what I was getting at it with my question, if I understand it correctly, this value can be any random string, as long as we get no collisions. So even time in nanoseconds would be enough. Probably with the "produced_resource" prefix, but that's it.

This is only used locally for FIFE.